src/main/java/org/venice/piazza/idam/controller/AuthController.java

```diff
@@ -371,4 +384,79 @@ else if (decodedAuthNInfo.split(":").length == 2) {
 			return new ResponseEntity<>(new ErrorResponse(error, IDAM_COMPONENT_NAME), HttpStatus.BAD_REQUEST);
 		}
 	}
+
+	@RequestMapping(value = "/login/geoaxis", method = RequestMethod.GET)
+	public RedirectView oauthRedirect() {
+	    final String redirectUri = oAuthClient.getRedirectUri(request);
```

Comment:
Consider adding some DEBUG or TRACE logging.

Reply:
Yeah, that's a good idea because I wasn't 100% sure it would give me what I was expecting.